source/MaterialXContrib/MaterialXMaya/CreateMaterialXNodeCmd.cpp

```diff
@@ -147,21 +143,37 @@ MStatus CreateMaterialXNodeCmd::doIt( const MArgList &args )
         MString documentFilePath;
         if (parser.isFlagSet(kDocumentFlag))
         {
-            CHECK_MSTATUS(argData.getFlagArgument(kDocumentFlag, 0, documentFilePath));
+            CHECK_MSTATUS(argData.getFlagArgument(kDocumentFlag, 0, documentFilePath))
         }
 
         if (documentFilePath.length() == 0)
         {
             throw mx::Exception("MaterialX document file path is empty.");
         }
 
-        mx::DocumentPtr document = MaterialXUtil::loadDocument(
-            documentFilePath.asChar(), Plugin::instance().getLibraryDocument()
-        );
+        mx::DocumentPtr document =
+            MaterialXUtil::loadDocument(documentFilePath.asChar(), Plugin::instance().getLibraryDocument());
 
         // Find renderables in the document
         std::vector<mx::TypedElementPtr> renderableElements;
-        mx::findRenderableElements(document, renderableElements);
+        std::vector<mx::TypedElementPtr> targetElements;
```

Comment:
Most of the changes are to sync with clang formatting.
This change and the OGSFragment change are the only 2 importance updates to support material nodes.